abrt.spec.in

```diff
@@ -281,25 +281,31 @@ Provides: libreport-plugin-bodhi = %{version}-%{release}
 Search for a new updates in bodhi server.
 %endif
 
-%package addon-python
+%package -n python2-abrt-addon
 Summary: %{name}'s addon for catching and analyzing Python exceptions
-Requires: python
 Requires: %{name} = %{version}-%{release}
 Requires: systemd-python
-Requires: abrt-python
+Requires: python2-abrt
+# Remove before F30
+Provides: abrt-addon-python = %{version}-%{release}
+Provides: abrt-addon-python%{?_isa} = %{version}-%{release}
+Obsoletes: abrt-addon-python < %{version}-%{release}
```

Comment:
Here should be last version-release+1 of abrt-addon-python
See https://fedoraproject.org/wiki/Packaging:Guidelines#Renaming.2FReplacing_Existing_Packages